eqasc/README.md

```diff
@@ -0,0 +1,24 @@
+[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/learning-to-explain-datasets-and-models-for/reasoning-chain-explanations-on-eqasc)](https://paperswithcode.com/sota/reasoning-chain-explanations-on-eqasc?p=learning-to-explain-datasets-and-models-for)
+
+This directory has code and data for the eQASC evaluator, as described in the EMNLP 2020 paper [Learning to Explain: Datasets and Models for Identifying Valid Reasoning Chains in Multihop Question-Answering](https://www.semanticscholar.org/paper/Learning-to-Explain%3A-Datasets-and-Models-for-Valid-Jhamtani-Clark/ebaebfefec9d5c21a4559a1a038743bd437d2f01).
+
+* [code](code/) holds the evaluator
+* [data/](data/) holds the ... TODO
+* [evaluator_data/eqasc](evaluator_data/eqasc/) holds the labels used by the evaluator
```

Comment:
Do you mind if I move these files to a simpler directory name, like `data`?

Reply:
sure sounds good

Reply:
Alrighty, I'll do this.